### DISCUSSION TITLE
How can one use an 'organizing principle' graph or domain knowledge graph to guide the extraction?

### DISCUSSION THREAD

**Discussion by sebastianschramm** [microsoft/graphrag#975] [Q&A]
Imagine I have some domain knowledge graph or organizing principle [(neo4j blog)](https://neo4j.com/blog/what-is-knowledge-graph/?utm_source=Google&utm_medium=PaidSearch&utm_campaign=Evergreen&utm_content=EMEA-Search-SEMCE-DSA-None-SEM-SEM-NonABM&utm_term=&utm_adgroup=DSA&gad_source=1&gclid=Cj0KCQjw2ou2BhCCARIsANAwM2Ejlgru3M6U9Fv_58Mhh8FAjXXF8BJAThYUBA_uKTACtmvDa-YzRykaAiUlEALw_wcB) as starting point. How can I use that graph to guide the extraction from my main dataset with graphrag such that my domain organizing principles are considered and reflected in the newly generated graph?

**Comment by Helenfool** [comment]
Hi, have you got a resolution? I have the same problem recently.
